client/main.lua

```diff
@@ -570,24 +575,22 @@ RegisterNetEvent('qb-doorlock:client:addNewDoor', function()
 				name = "checklock",
 				type = "checkbox",
 				options = {
-					{ value = "locked", text = Lang:t("general.locked_menu"), checked = (Config.SaveDoorDialog and doorData.locked) },
-					{ value = "pickable", text = Lang:t("general.pickable_menu"), checked = (Config.SaveDoorDialog and doorData.pickable == 'true') },
-					{ value = "cantunlock", text = Lang:t("general.cantunlock_menu"), checked = (Config.SaveDoorDialog and doorData.cantunlock == 'true') },
-					{ value = "hidelabel", text = Lang:t("general.hidelabel_menu"), checked = (Config.SaveDoorDialog and doorData.hidelabel == 'true') },
+					{ value = "locked", text = Lang:t("general.locked_menu"), checked = ((Config.SaveDoorDialog and doorData.locked) or false) },
+					{ value = "pickable", text = Lang:t("general.pickable_menu"), checked = ((Config.SaveDoorDialog and doorData.pickable and doorData.pickable == 'true') or false) },
+					{ value = "cantunlock", text = Lang:t("general.cantunlock_menu"), checked = ((Config.SaveDoorDialog and doorData.cantunlock and doorData.cantunlock == 'true') or false) },
+					{ value = "hidelabel", text = Lang:t("general.hidelabel_menu"), checked = ((Config.SaveDoorDialog and doorData.hidelabel and doorData.hidelabel == 'true') or false) },
 				}
 			}
 		}
 	})
 	if not dialog or not next(dialog) then canContinue = true return end
 	doorData = dialog
-
-	local identifier = doorData.configfile..'-'..doorData.dooridentifier
-	if Config.DoorList[identifier] then 
-		QBCore.Functions.Notify((Lang:t("error.door_identifier_exists")):format(identifier), 'error') 
-		canContinue = true 
-		return 
+	local identifier = doorData.dooridentifier
```

Comment:
```suggestion
        local identifier = doorData.configfile..'-'..doorData.dooridentifier
```
Config file entry is already set to required.